cmd/monaco/integrationtest/v2/config_restore_e2e_test.go

```diff
@@ -383,3 +383,14 @@ func cleanupDeployedConfiguration(t *testing.T, fs afero.Fs, manifestFilepath st
 
 	integrationtest.CleanupIntegrationTest(t, fs, manifestFilepath, loadedManifest, testSuffix)
 }
+
+func TestRestoreConfigs_FromDownloadWithPlatformManifestFile_withAutomation(t *testing.T) {
+	t.Setenv("MONACO_FEAT_AUTOMATION_RESOURCES", "true")
+	initialConfigsFolder := "test-resources/integration-download-configs-with-automation/"
+	manifestFile := initialConfigsFolder + "platform_manifest.yaml"
+	downloadFolder := "test-resources/download"
+	subsetOfConfigsToDownload := "alerting-profile,management-zone"
+	suffixTest := "_download_manifest"
```

Comment:
Please set a specific suffix for this test - suffix is appended to names/ids to keep configurations on the test envs from clashing. Currently this would overlap with the manifest download test resources. 
```suggestion
	suffixTest := "_download_automations"
```

Reply:
done